banzai/dbs.py

```diff
@@ -309,13 +329,17 @@ class SiteMissingException(Exception):
     pass
 
 
-def query_for_instrument(db_address, site, camera, enclosure, telescope, must_be_schedulable=False):
+def query_for_instrument(db_address, site, camera, enclosure=None, telescope=None,
```

Comment:
I would be hesitant to have `enclosure` and `telescope` default to `None`, since its fairly important for BANZAI imaging that they are specified, or you may end up returning the wrong instrument.

What if instead you relax the constraint on `enclosure` and `telescope` if `camera` contains the string `"nres"`?